docs/proposals/scheduling/20220629-schedule-device-in-card-level.md

```diff
@@ -0,0 +1,268 @@
+---
+title: schedule device(gpu\rdma) in card level
+authors:
+- "@buptcozy"
+reviewers:
+- "@hormes"
+- "@allwmh"
+- "@honpey"
+- "@saintube"
+- "@stormgbs"
+- "@zwzhang0107"
+- "@eahydra"
+creation-date: 2022-06-29
+last-updated: 2022-06-29
+status: provisional
+
+---
+
+# schedule device in card-level
+
+## Table of Contents
+
+<!--ts-->
+
+* [schedule device in card level](#schedule-device-in-card-level)
+  * [Table of Contents](#table-of-contents)
+  * [Summary](#summary)
+  * [Motivation](#motivation)
+  * [Goals](#goals)
+  * [User Stories](#user-stories)
+    * [Story NVLink](#story-NVLink)
+    * [Story MIG](#story-MIG)
+    * [Story Oversold](#story-Oversold)
+  * [Implementation Details](#implementation-details)
+    * [scheduler](#scheduler)
+    * [device reporter](#device-reporter)
+    * [RuntimeProxy](#runtimeProxy)
+
+<!--te-->
+
+## Summary
+support schedule devices like gpu\rdma\fpga in card level for better schedule placement. 
+
+## Motivation
+there is lots of cases that need scheduler make better schedule placement with devices like gpu\rdma etc.
+like gpu-nvlink in A100\V100、gpu Mig in A100.
+
+### Goals
+1. a device reporter per node will report card level device crd, which will not conflict with kubelet and existed device plugin
+
+2. scheduler schedule pod with device card level
+
+3. runtime-proxy use scheduler device schedule result to overwrite kubelet\device-plugin card assign result
+
+### User Stories
+
+#### Story NVLink
+NVLink
+
+![image](/docs/images/nvlink.jpg)
+
+through the picture we can find that although there's 8 card in A100\V100 node, the data-transport-speed between cards 
+are not same. when a pod need multi-gpu-cards, we have chance to select a better placement with largest data-transport-speed. 
+
+#### Story MIG
+![image](/docs/images/mig.jpg)
+
+A100 gpu card support mig function to split a card into different size of ci\gi, which provide isolate ability.
```

Comment:
No design details about the MIG are seen in this Proposal. If missing, please add details; if not, it is recommended to submit a new proposal separately.

Reply:
In this case, it actually has nothing to do with this PR and can be removed directly.

Reply:
deleted